pingcap/tidb-operator
# remove br binary in tidb-backup-manager image

### What problem does this PR solve? 
remove BR binary in the Dockerfile.

fix #3526 

### What is changed and how does it work?

### Check List 

Tests 

- Manual test (add detailed scripts or steps below)
    - create a `Backup` CR such as `manifests/backup/backup-aws-s3-br.yaml`
    - check that everything is fine, nothing breaks

Code changes

None

Side effects

None

Related changes

 - Need to cherry-pick to the release branch

### Does this PR introduce a user-facing change?:

```release-note
NONE
```
